kombu/utils/compat.py

```diff
@@ -82,7 +82,12 @@ def entrypoints(namespace):
     if sys.version_info >= (3,10):
         entry_points = importlib_metadata.entry_points(group=namespace)
     else:
-        entry_points = importlib_metadata.entry_points().get(namespace, [])
+        entry_points = importlib_metadata.entry_points()
+        try:
+            entry_points = entry_points.get(namespace, [])
+        except AttributeError:
+            entry_points = entry_points.select(group=namespace)
```

Comment:
This change requires `importlib_metadata >= 3.6`.

Reply:
done